tiny_dnn/util/parameter_init.h

```diff
@@ -0,0 +1,64 @@
+/*
+    Copyright (c) 2013, Taiga Nomi and the respective contributors
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
```

Comment:
https://github.com/alykhantejani/nninit/blob/master/nninit.py
just for reference and future work, you might add a TODO with the link